Cargo.toml

```diff
@@ -13,11 +13,16 @@ tokio = { version = "1.33.0", features = ["full"] }
 serde = "1.0.183"
 serde_path_to_error = "0.1.14"
 serde_json = "1.0.107"
-ethereum-types = "0.14.1"
 thiserror = "1.0.50"
 futures = "0.3.29"
-rlp = "0.5.2"
 keccak-hash = "0.10.0"
+alloy = { git = "https://github.com/alloy-rs/alloy", features = [
```

Comment:
Does anyone happen to know why they haven't/aren't releasing this? It makes me slightly uncomfortable with unreleased code but I don't think we really have another option here. We've been looking at `alloy` in the agglayer too but haven't made the switch yet for this exact reason. But... I think we just bite the bullet here.